lib/streamlit/connections/sql_connection.py

```diff
@@ -43,31 +43,27 @@
 
 
 class SQLConnection(ExperimentalBaseConnection["Engine"]):
-    """A connection to a SQL database using a SQLAlchemy Engine.
-    Initialize using ``st.experimental_connection("<name>", type="sql")``.
+    """A connection to a SQL database using a SQLAlchemy Engine. Initialize using ``st.experimental_connection("<name>", type="sql")``.
 
     SQLConnection provides the ``query()`` convenience method, which can be used to
     run simple read-only queries with both caching and simple error handling/retries.
-
     More complex DB interactions can be performed by using the ``.session`` property
     to receive a regular SQLAlchemy Session.
 
-    SQLConnections will typically use connection parameters specified via
-    ``st.secrets`` or ``**kwargs``. SQLConnections should always be created using
-    ``st.experimental_connection()``, **not** initialized directly.
-
-    Particular parameters that may be frequently used:
+    SQLConnections should always be created using ``st.experimental_connection()``,
+    **not** initialized directly. SQLConnections commonly specify connection parameters
```

Comment:
Hm, I think the "SQLConnections commonly specify connection parameters" sentence reads a bit weirdly and should instead be changed to something like "Connection parameters for a SQLConnection can be specified using either..."

Will go ahead and make a commit making this change